buildroot-external/configs/odroid_m1_defconfig

```diff
@@ -5,7 +5,7 @@ BR2_CCACHE=y
 BR2_CCACHE_DIR="/cache/cc"
 BR2_OPTIMIZE_2=y
 BR2_ENABLE_LTO=y
-BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_HASSOS_PATH)/patches $(BR2_EXTERNAL_HASSOS_PATH)/board/hardkernel/patches $(BR2_EXTERNAL_HASSOS_PATH)/board/hardkernel/odroid-m1/patches"
+BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_HASSOS_PATH)/patches $(BR2_EXTERNAL_HASSOS_PATH)/board/hardkernel/odroid-m1/patches"
```

Comment:
There are patches in this directory, but I see these are really not needed for M1. Nice catch!

Reply:
On the other hand, we normally pick all the patches from top to the bottom, so this will be an exception now. Maybe we should consider using a different folder for platform-specific patchsets?

Reply:
Most others dont even have $(vendor)/patches directory, except for Raspberry!


All of the patches in `hardkernel/patches/linux`  seem to be board specific DT patches for C2/C4/N2, there is nothing there I can see that is platform specific, perhaps they could just be moved to relevant board folders? I guess the uboot patch probably applies to any amlogic board though.

I suppose there would be a case for a shared `rockchip-patches` folder given Green/M1/M1S likely have some common patches, for example the RNG patcheset which seems to have died upstream. And on that note perhaps also a shared `kernel-rockchip.config` given they both identical

Reply:
Sure, I will shuffle some things around.